dist/install.tmpl

```diff
@@ -8,16 +8,16 @@ set -Eeuo pipefail
 
 INSTALL_DIR="$(dirname "$0")"
 
-hub="$INSTALL_DIR/hub"
+file="$INSTALL_DIR/{{ .Env.TXTL_FILE }}"
```

Comment:
This is now a template file used by Gomplate

Reply:
Slick